### ISSUE TITLE
Suggestions for Web App Manifest

### ISSUE THREAD

**Issue by filips123** [cookiengineer/stealth-browser#12]
I have some suggestions for Web App Manifest (currently [`manifest.json`](https://github.com/cookiengineer/stealth/blob/X0/browser/manifest.json)):

* It should be renamed to `site.webmanifest`.
  Although it is not required, it is [reccomended by W3C](https://w3c.github.io/manifest/#media-type-registration) that manifest uses ` application/manifest+json` media type and `.webmanifest ` extension.

* It should use `"display": "standalone"`.
  "The application will look and feel like a standalone application. This can include the application having a different window, its own icon in the application launcher, etc. In this mode, the user agent will exclude UI elements for controlling navigation, but can include other UI elements such as a status bar." (from MDN)

* It's `short_name` should probably be `Stealth` (with upper first letter).

* Some additional icons may be added. You can use [Real Favicon Generator](https://realfavicongenerator.net/) for this.

**Comment by cookiengineer** [comment]
I pushed the Fixes to the https://github.com/cookiengineer/stealth/tree/X0 branch, and double-checked with the official IANA registrations for [media types](https://www.iana.org/assignments/media-types/media-types.xhtml) list.

As it's not officially registered yet, I added a non-official section to the URL parser.
I generated some favicons and chose the theme color as far as possible, but it still has to be evaluated (currently have no Android Tablet to verify behaviour).
